package.json

```diff
@@ -28,7 +28,7 @@
     "nopt": "2 || 3",
     "npmlog": "0 || 1 || 2 || 3 || 4",
     "osenv": "0",
-    "request": "2",
+    "request": "^2.86.0",
```

Comment:
Although the version should match, maybe use 2.87.0 since they removed the heok/hawk issues https://github.com/request/request/blob/master/CHANGELOG.md#v2870-20180521